src/astarte-client/client.ts

```diff
@@ -910,6 +910,17 @@ astarteAPIurl`${config.realmManagementApiUrl}v1/${'realm'}/interfaces/${'interfa
       data,
     );
   }
+  async deleteDataFromInterface(params: {
```

Comment:
While this technically deletes data, it's true function is to unset a property.
Let's call it accordingly
```suggestion
  async unsetProperty(params: {
```